Verovio.podspec

```diff
@@ -1,21 +1,21 @@
 Pod::Spec.new do |s|
   s.name           = 'Verovio'
-  s.version        = '4.1.0-dev'
-  s.license        = { :type => 'LGPL' }
+  s.version        = '4.0.1'
```

Comment:
keep `4.1.0-dev`

Reply:
No, that will not work. The pod linter checks out the repo and builds it locally, and there is no `4.1.0-dev` tag.

Reply:
No, it will be push from `master` with `4.1.0` then and there will be a corresponding.